SU-SWS/stanford_capx
# CAPX-6 | CAPx Management Menu Block + Placement

# Ready for review
- Creates and places a management menu block on each capx admin page.
### Test
- Navigate to admin/config/capx
- See menu in the top right portion of the content region
- Click on each link and ensure menu stays there.
